### ISSUE TITLE
"No Hunger Restoration" Option on /cmi heal

### ISSUE THREAD

**Issue by VolcanoMan3705** [Zrips/CMI#4321]
Add an extra argument to /cmi heal that specifies true or false on feeding the player, defaulting to true.

**Secondary Option** Add an extra argument to /cmi heal that specifies how much the heal will feed the target. Defaults to 20. Can go negative.

**Detailed explanation of what it should do, when and in what situation:** All it will do is serve as a Boolean for whether or not the player being affected by /cmi heal will have their hunger restored. Or the secondary option listed, which is to let people specify how much to feed the target. I imagine the format would be something like this ---

/cmi heal (healamount/healpercent) feed:[true/false]
or
/cmi heal (healamount/healpercent) (feedamount)

**Basic explanation as to why this feature is needed and/or useful to have:** I actually use the /cmi heal as passive health regeneration and for various other healing abilities and for taking health away as well, so it restoring hunger every time it is used is actually a very negative side effect to this.

**Comment by CloudeLecaw** [comment]
You can create a custom alais that first heal the target and  then set hunger. You can combine this to an command

**Comment by TheExceed** [comment]
@CloudeLecaw So, I just tested this as I was interested in this too. 
What the op is asking for is for /heal to be able to not feed.
By default, /cmi heal will heal the player AND feed.

Example: /heal
https://gyazo.com/46b7820d79b7d396a9af15519dc34a15

Example: /hheal (from your code but removing the hunger code
https://gyazo.com/21249f215539c7488db03a96b4ee05bb

`  hheal:
    Cmds:
    - perm:cmi.command.heal! asConsole! cmi heal $1 -s
    - perm:cmi.command.heal! msg! $1 &eYou got &6healed &eand your hunger has been restored to&6 $2 hunger&e.
    - perm:cmi.command.heal! msg! [playerName] &eYou healed&6 $1 &eand restored his hunger to&6 $2 hunger&e. `

**Comment by TheExceed** [comment]
@CloudeLecaw So, I just tested this as I was interested in this too.
What the op is asking for is for /heal to be able to not feed.
By default, /cmi heal will heal the player AND feed.

Example: /heal
https://gyazo.com/46b7820d79b7d396a9af15519dc34a15

Example: /hheal (from your code but removing the hunger code
https://gyazo.com/21249f215539c7488db03a96b4ee05bb


 ```
  hheal:
    Cmds:
    - perm:cmi.command.heal! asConsole! cmi heal $1 -s
    - perm:cmi.command.heal! msg! $1 &eYou got &6healed &eand your hunger has been restored to&6 $2 hunger&e.
    - perm:cmi.command.heal! msg! [playerName] &eYou healed&6 $1 &eand restored his hunger to&6 $2 hunger&e. 
```

Also, tried adding `hungerrestore` to the RemoveNegative list. Did not remove hunger
```
heal:
  RemoveNegative:
    # Do you want to remove negative potion effects from player on heal
    use: true
    List:
    - blindness
    - confusion
    - harm
    - hunger
    - poison
    - slow
    - slow_digging
    - weakness
    - wither
    - hungerrestore```

**Comment by VolcanoMan3705** [comment]
Yeah, @TheExceed got the intent correct. Basically, I added the "**Second Option**" as a way to increase the flexibility of /cmi heal rather than only setting "FEED" to true/false, but I didn't think of creating custom aliases to add food as a way to achieve the second option.

So basically, the issue is not controlling how much to feed the player, but rather just not having **/cmi heal** feed the player at all.

**TL:DR: True/False option for feeding player with /cmi heal**

**Comment by Zrips** [comment]
**/cmi heal [playerName/all] (healamount/healpercent) (-nofeed) (-ignoreffects) (-dontextinguish)**
Next update

**Comment by CloudeLecaw** [comment]
Wonder full now just a -s is missing🙃
